### ISSUE TITLE
Errors on fresh install and start

### ISSUE THREAD

**Issue by rogerballard** [truffle-box/drizzle-box#29]
Hi, I'm having some trouble running this box for the first time. I have followed the steps as described in the readme, however I can't access the homepage for the box at all. Hopefully I'm doing something stupid, but below are the steps I have followed.

Running on MacOS
Node version: `8.5.0`
NPM version: `5.6.0`

----

Fresh install and compile
```javascript
npm i -g truffle
npm i -g ganache-cli

truffle unbox drizzle

ganache-cli -b 3 // in a separate window

truffle compile
truffle migrate
```

Start the development server
```javascript
npm run start
```

Encounter issue https://github.com/truffle-box/drizzle-box/issues/24, so follow the steps to fix as described in the comments:

```javascript
npm update

+ drizzle-react@1.1.0
+ drizzle@1.1.3
+ zeppelin-solidity@1.8.0
+ react-router@3.2.1
+ drizzle-react-components@1.1.0

truffle compile
truffle migrate

npm start
```

At this point, http://localhost:3000 is still erroring:
<img width="1159" alt="screen shot 2018-04-22 at 13 25 56" src="https://user-images.githubusercontent.com/12383481/39094987-c6c3eb2e-4630-11e8-96e8-e9bfd2ec201a.png">

Metamask is connected to port 8545, has the first account imported via private key, and is unlocked.

**Comment by techbubble** [comment]
I am having the exact same issue with identical console errors. I tracked down the line in drizzle.js where it's blowing up, but haven't found a resolution.

![error](https://user-images.githubusercontent.com/470750/39097939-2251bdf4-4618-11e8-9116-3c06ffcef75d.png)

**Comment by pemulis** [comment]
I'm experiencing this issue, too. Digging into Drizzle code and documentation to see if I can learn more but doubt I'll be the first to solve the problem.

**Comment by brtknr** [comment]
Yes, I had that too but I manually installed it using npm install and that went away but there was still the problem posted on this thread!

**Comment by techbubble** [comment]
@ColdDevil that worked for me. Thank you.

**Comment by pemulis** [comment]
Worked for me, too! Thanks @ColdDevil!

**Comment by LCarballoBaseCodeIt** [comment]
As @ColdDevil said. The box needs those packages too:
"eth-block-tracker-es5": "^2.3.2"
"web3": "1.0.0-beta.34"
